shared/version.js

```diff
@@ -0,0 +1,14 @@
+const versionRe = /(?:R2Northstar\/)(\d)\.(\d)\.(\d)/g
```

Comment:
Should be

```suggestion
const versionRe = /(?:R2Northstar\/)(\d+)\.(\d+)\.(\d+)/g
```

to allow capturing version numbers with more than one digit like e.g. `R2Northstar/1.23.4`.

Especially as https://github.com/R2Northstar/NorthstarLauncher/pull/113 in its current form spits out version numbers with multiple digits for release candidate versions.

Reply:
good point